CDT/include/Triangulation.hpp

```diff
@@ -1093,6 +1099,12 @@ void Triangulation<T, TNearPointLocator>::insertVertex(const VertInd iVert)
     const VertInd walkStart = m_nearPtLocator.nearPoint(v, vertices);
     insertVertex(iVert, walkStart);
```

Comment:
This will also call insert-vertex callback, so the callback will be called twice. Is it intentional?